API.md

```diff
@@ -0,0 +1,269 @@
+# API
+
+* [`create`](#create)
```

Comment:
This is a static function but the rest are instance methods however they appear the same here. Might be a good idea to differentiate :)